### PR TITLE
docs(providers): Added documentation for missing providers

### LINKED ISSUE #2033: [📃 Docs]: Missing documentation for a few providers
Those providers are missing docs:

```
"service now",
"python",
"prometheus",
"gke",
"github",
"dynatrace",
"kafka",
"auth0",
"bash",
"keep",
"bigquery",
"parseable",
```

Python and bash are disabled for cloud, "keep" provider is a placeholder for alerts without a provider.

### PR DESCRIPTION
**Pull request by rishisulakhe**
[Fixes #2033]
/claim #2033

Closes #2033 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [X] My code requires changes to the documentation
- [X ] I have updated the documentation as required
- [ X] All the tests have passed

### CODE REVIEW

**Comment by talboren** [comment]
Please adjust the PR name to follow conventions 🙏🏼 thanks for the contribution!

File: docs/providers/documentation/service-now-provider.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Service Now"
+sidebarTitle: "Service Now Provider"
+description: "Service Now provider allows sending notifications and updates to Service Now."
+---
```
Comment (talboren) [inline docs/providers/documentation/service-now-provider.mdx:5]: Hey @rishisulakhe, thank you so much for this!
ServiceNow provider also allows one to get the topology information from ServiceNow CMDB. Can you adjust the documentation to include that as well?

File: docs/providers/documentation/keep-provider.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Keep"
+sidebarTitle: "Keep Provider"
+description: "Keep provider acts as a placeholder for alerts without a specific provider."
```
Comment (talboren) [inline docs/providers/documentation/keep-provider.mdx:4]: It actually allows you to query the alerts you have in Keep, can we adjust that?

File: docs/providers/documentation/bash-provider.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (talboren) [inline docs/providers/documentation/bash-provider.mdx:1]: Also bash provider has a limitation of cloud execution, it is part of the limited providers.

File: docs/providers/documentation/auth0-provider.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Auth0"
+sidebarTitle: "Auth0 Provider"
+description: "Auth0 provider allows integration with Auth0 for user authentication and authorization."
```
Comment (talboren) [inline docs/providers/documentation/auth0-provider.mdx:4]: This is actually inaccurate. Can you please adjust?

**Review by talboren — CHANGES_REQUESTED** [review]
Thank you so much for this @rishisulakhe! I left a few comments and look forward to merging this into main once you fix them :)

File: docs/providers/documentation/service-now-provider.mdx (hunk 1 above)
Comment (rishisulakhe) [reply docs/providers/documentation/service-now-provider.mdx:5]: Yep, will add this

File: docs/providers/documentation/keep-provider.mdx (hunk 2 above)
Comment (rishisulakhe) [reply docs/providers/documentation/keep-provider.mdx:4]: Will add this also

File: docs/providers/documentation/bash-provider.mdx (hunk 3 above)
Comment (rishisulakhe) [reply docs/providers/documentation/bash-provider.mdx:1]: Yep, will add this

File: docs/providers/documentation/auth0-provider.mdx (hunk 4 above)
Comment (rishisulakhe) [reply docs/providers/documentation/auth0-provider.mdx:4]: Yep, will  update it

**Comment by rishisulakhe** [comment]
Hello, @talboren 
I have addressed the feedback and made the necessary changes. Please review the latest updates.

**Review by talboren — APPROVED** [review]
LGTM

**Comment by talboren** [comment]
@rishisulakhe please fix https://github.com/keephq/keep/actions/runs/11315408736/job/31466554811?pr=2167 so we can merge this.

cc: @Matvey-Kuk

**Comment by rishisulakhe** [comment]
Hey, @talboren
I have done with necessary changes. Please review the latest updates.

File: docs/providers/documentation/bash-provider.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+## Authentication Parameters
+
+_None required for local execution of Bash scripts._
+
+## Connecting with the Provider
+
+The Bash provider allows you to run Bash commands or scripts in your workflow. You can pass in any valid Bash command, and it will be executed in a local environment. 
+
+### **Cloud Limitation**
+This provider is disabled for cloud environments and can only be used in local or self-hosted environments.
```
Comment (Matvey-Kuk) [inline docs/providers/documentation/bash-provider.mdx:26]: <3

File: docs/providers/documentation/auth0-provider.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (Matvey-Kuk) [inline docs/providers/documentation/auth0-provider.mdx:1]: Please mention that if the user is looking for Auth0 as an auth method for keep, they should refer to https://docs.keephq.dev/deployment/authentication/auth0-auth

**Comment by Matvey-Kuk** [comment]
Looks great! Please fix the minor comment above and I'll merge!

File: scripts/docs_get_providers_list.py [hunk 7]
```diff
@@ -15,18 +15,7 @@
...
-    "gke",
-    "github",
-    "dynatrace",
-    "kafka",
-    "auth0",
-    "bash",
-    "keep",
-    "bigquery",
-    "parseable",
+    "smtp"
```
Comment (Matvey-Kuk) [inline scripts/docs_get_providers_list.py:18]: Please remove SMTP as well.

Comment (rishisulakhe) [reply scripts/docs_get_providers_list.py]: Done

**Comment by rishisulakhe** [comment]
@Matvey-Kuk 
Done with all necessary changes.
Thanks

**Review by Matvey-Kuk — APPROVED** [review]
Great, thank you!

File: docs/providers/documentation/gke-provider.mdx [hunk 8]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Google Kubernetes Engine (GKE)"
```
Comment (Matvey-Kuk) [inline docs/providers/documentation/gke-provider.mdx:2]: Looks like title is making tests to fail: https://github.com/keephq/keep/actions/runs/11316619299/job/31469161221?pr=2167
```suggestion
title: "GKE" 
```

File: docs/providers/documentation/parseable-provider.mdx [hunk 9]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Bash"
```
Comment (Matvey-Kuk) [inline docs/providers/documentation/parseable-provider.mdx:2]: Looks like a mistake, it's parseable-provider.mdx 0_o

**Review by Matvey-Kuk — CHANGES_REQUESTED** [review]

File: docs/providers/documentation/parseable-provider.mdx (hunk 9 above)
Comment (rishisulakhe) [reply docs/providers/documentation/parseable-provider.mdx:2]: Shit, my mistake

Comment (Matvey-Kuk) [reply docs/providers/documentation/parseable-provider.mdx:2]: Lol, no worries!

File: docs/providers/documentation/gke-provider.mdx (hunk 8 above)
Comment (rishisulakhe) [reply docs/providers/documentation/gke-provider.mdx]: updated

**Comment by rishisulakhe** [comment]
@Matvey-Kuk
Done

**Comment by Matvey-Kuk** [comment]
@rishisulakhe I see tests are still failing

**Comment by rishisulakhe** [comment]
@Matvey-Kuk 
Fixes the last error

File: package-lock.json [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "keep",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```
Comment (Matvey-Kuk) [inline package-lock.json:6]: Why do we need this?

File: docs/mint.json [hunk 11]
```diff
@@ -183,7 +183,19 @@
...
+"providers/documentation/bigquery-provider",
+"providers/documentation/dynatrace-provider",
+"providers/documentation/github-provider",
+"providers/documentation/gke-provider",
+"providers/documentation/kafka-provider",
+"providers/documentation/keep-provider",
+"providers/documentation/parseable-provider",
+"providers/documentation/prometheus-provider",
+"providers/documentation/python-provider",
+"providers/documentation/service-now-provider"
```
Comment (Matvey-Kuk) [inline docs/mint.json:198]: Formatting is odd here

**Review by Matvey-Kuk — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 10 above)
Comment (rishisulakhe) [reply package-lock.json]: It was added unintentionally. Will delete this file.

File: package-lock.json [hunk 12]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "docs",
+  "name": "keep",
```
Comment (talboren) [inline package-lock.json:2]: still some strange change here. why do you change the location of the file?

**Comment by rishisulakhe** [comment]
@Matvey-Kuk 
It's ready for review. Please let us know if you have any concerns.

File: docs/mint.json [hunk 13]
```diff
@@ -183,7 +183,19 @@
...
+            "providers/documentation/bigquery-provider",
+            "providers/documentation/dynatrace-provider",
+            "providers/documentation/github-provider",
+            "providers/documentation/gke-provider",
+            "providers/documentation/kafka-provider",
+            "providers/documentation/keep-provider",
+            "providers/documentation/parseable-provider",
+            "providers/documentation/prometheus-provider",
+            "providers/documentation/python-provider",
+            "providers/documentation/service-now-provider"
```
Comment (Matvey-Kuk) [inline docs/mint.json:198]: I'm sorry to add one more comment, but the list was alphabetically formatted before. Kindly asking to review all changes before publishing them, thank you.

**Review by Matvey-Kuk — CHANGES_REQUESTED** [review]

File: docs/mint.json (hunk 13 above)
Comment (rishisulakhe) [reply docs/mint.json:198]: No worries, will do this also

File: package-lock.json (hunk 12 above)
Comment (rishisulakhe) [reply package-lock.json]: unintentionally, it was done

File: package-lock.json
Comment (talboren) [inline package-lock.json:1]: Still file rename. Can we undo this?

Comment (rishisulakhe) [reply package-lock.json:1]: I have changed name from "keep" to "docs". What else i can do?

Comment (Matvey-Kuk) [reply package-lock.json:1]: Why did you move the file?

File: docs/mint.json [hunk 14]
```diff
@@ -107,84 +107,97 @@
...
+              "providers/documentation/mock-provider",
+              "providers/documentation/mongodb-provider",
+              "providers/documentation/mysql-provider",
+              "providers/documentation/netdata-provider",
+              "providers/documentation/new-relic-provider",
+              "providers/documentation/ntfy-provider",
+              "providers/documentation/openobserve-provider",
+              "providers/documentation/openshift-provider",
+              "providers/documentation/opsgenie-provider",
+             "providers/documentation/pagerduty-provider",
```
Comment (Matvey-Kuk) [inline docs/mint.json:161]: Why is it formatted so?

**Review by Matvey-Kuk — CHANGES_REQUESTED** [review]

File: docs/mint.json (hunk 14 above)
Comment (rishisulakhe) [reply docs/mint.json]: Corrected.

**Review by talboren — APPROVED** [review]
LGTM

**Review by Matvey-Kuk — APPROVED** [review]

**Comment by rishisulakhe** [comment]
@Matvey-Kuk @talboren 
Thanks for helping me.
This was my first PR that has been merged.
Will contribute more to "keep" in future.

**Comment by Matvey-Kuk** [comment]
Thank you @rishisulakhe !

**Comment by rishisulakhe** [comment]
@Matvey-Kuk How to claim bounty?

**Comment by Matvey-Kuk** [comment]
@rishisulakhe I talk to the Algora, their bot should popup here with instructions, but it didn't for some reasons.

**Comment by Matvey-Kuk** [comment]
@rishisulakhe while we're dealing with Algora (I'm not sure if we'll be able to return the bot), may I ask you to enable GitHub Sponsorship for your account? We'll use it to compensate the contribution.

**Comment by rishisulakhe** [comment]
@Matvey-Kuk
 I have just enabled Github Sponsored for my account. It is pending for approval.

**Comment by rishisulakhe** [comment]
@Matvey-Kuk @talboren  
My github Sponsers profile is now live. Now you can use it to compensate my contribution. 
Thanks

**Comment by Matvey-Kuk** [comment]
@rishisulakhe on it!

**Pull request merged**